app/feedme/analytics/performance_monitor.py

```
@@ -0,0 +1,835 @@
+                total_count = len(hits) if hits else 0
+                
+                cache_hits.append(hit_count)
+                total_requests.append(total_count)
+            
+            total_hits = sum(cache_hits)
+            total_reqs = sum(total_requests)
+            
+            hit_rate = total_hits / total_reqs if total_reqs > 0 else 0
+            
+            # Calculate average response times with/without cache
+            response_times_with_cache = []
```

Ruff (0.11.9)

Local variable `response_times_with_cache` is assigned to but never used

Remove assignment to unused variable `response_times_with_cache`

(F841)